test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -335,4 +336,68 @@ def add_locations
     assert_response :success
     assert_equal 1, location.reload.location_parameters.count
   end
+
+  test "should create with comma separated name" do
+    name = "%s, %s" % [RFauxFactory.gen_alpha, RFauxFactory.gen_alpha]
+    post :create, params: { :location => { :name => name } }
+    assert_response :success
+    show_response = JSON.parse(@response.body)
+    assert !show_response.empty?
+  end
+
+  test "should create with description" do
+    post :create, params: { :location => { :name => RFauxFactory.gen_alpha,  :description => RFauxFactory.gen_alpha } }
```

Comment:
avoid double spaces before `:description` key, rubocop (ruby linter) would not be happy